Update README.md with changesets description (#1094)
guardian/csnx: README.md

To run Chromatic (for visual regression testing) on a PR, add the `run_chromatic` label to the PR in Github. The Chromatic tests are required to pass by CI, so this will _need_ to be done at least once before a PR is merged. We recommend you only add the label once you are happy with any changes made and have checked for visual regressions manually first.

### Releasing a package

Libs within CSNX are available as NPM packages. We use [Changesets](<[url](https://github.com/changesets/changesets)>) to automate this release process.

> If you wish to do an NPM release, run `make changeset`. This open the Changesets CLI, and you will be offered a list of packages to release. Once you've selected the desired package/s you'll be given the option of `major`, `minor` or `patch` release. Select one of these and add a description.

Reviewer: just a suggestion, feel free to reject it

```suggestion
To publish your changes to NPM, run `make changeset`. This open the Changesets CLI, and you will be offered a list of packages to release. Once you've selected the changed package/s you'll be given the option of `major`, `minor` or `patch` release. Select one of these and add a description.
```
